.appveyor.yml

```diff
@@ -0,0 +1,44 @@
+platform:
+  - x64
+
+branches:
+  only:
+  - master
+
+environment:
+
+  matrix:
+    - TARGET_ARCH: x86
+      PYVER: 27
+      CONDA_INSTALL_LOCN: C:\\Miniconda
+
+    - TARGET_ARCH: x64
+      PYVER: 27
+      CONDA_INSTALL_LOCN: C:\\Miniconda-x64
+
+    - TARGET_ARCH: x86
+      PYVER: 35
+      CONDA_INSTALL_LOCN: C:\\Miniconda35
+
+    - TARGET_ARCH: x64
+      PYVER: 35
+      CONDA_INSTALL_LOCN: C:\\Miniconda35-x64
+
+    - TARGET_ARCH: x86
+      PYVER: 36
+      CONDA_INSTALL_LOCN: C:\\Miniconda36
+
+    - TARGET_ARCH: x64
+      PYVER: 36
+      CONDA_INSTALL_LOCN: C:\\Miniconda36-x64
+
+install:
```

Comment:
`conda-forge` also removes Cygwin before installing `conda` due to some historical issues with Cygwin's `git`. We did not encounter those issues here. So skipped that step. However, should it be needed, we simply need to run `rmdir C:\cygwin /s /q`.